selectors/emojis.js

```diff
@@ -0,0 +1,16 @@
+// Copyright (c) 2017 Mattermost, Inc. All Rights Reserved.
+// See License.txt for license information.
+
+import {createSelector} from 'reselect';
+
+import {getCustomEmojisByName} from 'mattermost-redux/selectors/entities/emojis';
+
+import {EmojiMap} from 'stores/emoji_store.jsx';
+
+export const getEmojiMap = createSelector(
+    getCustomEmojisByName,
+    (customEmojisByName) => {
+        console.log('recalculating emojis1');
```

Comment:
Why the log?

Reply:
Oops, that was for testing. That's also concerning that that wasn't caught by the style checker

Reply:
@hmhealey tell me about it!